<launch>
 <node pkg="ldlidar" type="ldlidar" name="ldlidar_ssl20"  output="screen">
  <param name="product_type" type="str" value="SSL20P"/>
  <param name="port_name" type="str" value="/dev/ttyACM0"/>
  <param name="frame_id" type="str" value="base_laser"/>
  <param name="topic_name" type="str" value="scan"/>
 </node>
<!-- publisher tf transform, parents frame is base_link, child frame is base_laser -->
<!-- args="x y z yaw pitch roll parents_frame_id child_frame_id period_in_ms"-->
  <node name="base_to_laser" pkg="tf" type="static_transform_publisher"  args="0.0 0.0 0.18 0 0.0 0.0 base_link base_laser 100"/>
</launch>